### ISSUE TITLE
how do i use train for single voice

### ISSUE THREAD

**Issue by wac81** [anonymous-pits/pits#15]
how do i use train for single voice

**Comment by anonymous-pits** [comment]
You just need to change [configs/config_en.yaml's speakers](https://github.com/anonymous-pits/pits/blob/57ca2ef3778396f0f15cd5f186a30925a5e8795e/configs/config_en.yaml#LL37C1-L38C1) to single element list as ["spk0"].

**Comment by wac81** [comment]
yes, i change this place ,and change sr to 22050,but get some error in inference for few sentences.

torch shape is not match.
